coordinator/scripts/luigi_scoop_server.py

```diff
@@ -0,0 +1,206 @@
+#! /usr/bin/env python
+
+import roslib
+roslib.load_manifest('coordinator')
+import rospy
+import actionlib
+from std_msgs.msg import *
+import sched, time
+
+# Imported msgs are OrderIceCreamAction, OrderIceCreamActionFeedback
+# OrderIceCreamActionGoal, OrderIceCreamActionResult, OrderIceCreamFeedback 
+# OrderIceCreamGoal, OrderIceCreamResult
+from roboy_cognition_msgs.msg import *
+
+from roboy_control_msgs.srv import TranslationalPTPMotion
+from geometry_msgs.msg import Point
+
+import numpy as np
+
+# First one is left box, second one is right box
+availableFlavors = ['choco', 'spagetti']
+
+# The number of steps we need to scoop a single scoop (tuned manually)
+scoopingSteps = 5
+
+class ScoopServer:
+  _feedback = OrderIceCreamFeedback()
+  _result = OrderIceCreamResult()
+  _feedback_delay = 5
+
+  def __init__(self):
+    self._action_name = 'luigi_scoop'
+    self.doneScooping_ = False
+    self._feedback.finished_scoops = []
+
+    # Status as to be sent for luigig as a feedback
+    self.scooping_human_status_ = 'Did not receive any ice cream order yet!'
```

Comment:
I think we don't need this. Luigi only expects feedback after the action has been started.

Reply:
Its okay, it helps us to understand what is happening , e.g. printing it while debugging